content/cloud/changelog/spin-cloud-v0-5-0.md

```diff
@@ -0,0 +1,28 @@
+title = "Spin Cloud v0.5 Updates"
+template = "changelog_item"
+date = "2023-11-02T12:00:00Z"
+enable_shortcodes = true
+tags = ["spin_cloud_plugin"]
+[extra]
+type= "changelog_post"
+twitter_card_type = "summary_large_image" 
+image = "/static/image/changelog/spin-cloud-0.5.0.jpg"
+
+---
+
+## Spin Cloud v0.5.0 Changelog
```

Comment:
During final v2.0 release prep today, we discovered we needed to bump spin crates in cloud plugin to get a few fixes, hence now the latest release is [v0.5.1](https://github.com/fermyon/cloud-plugin/releases/tag/v0.5.1).  Technically all of the user-facing changes mentioned below _are_ in v0.5.0 so perhaps all of this is fine to leave as is.  Users will automatically get v0.5.1 when they upgrade to or install Spin v2.0.0, though.